content/app-dev/create-provider/testing.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Testing the New Provider"
+date: 2018-12-29T11:02:05+06:00
+weight: 10
+draft: false
+---
+
+The easiest way to test your new capability provider is as follows:
+
+1. Start up a new local OCI registry (we have an example of these `docker-compose` files in multiple sample repositories)
+
+1. Upload the newly-created _provider archive_ to the local OCI registry (You can use `wash reg push`)
+
+1. Upload an actor that utilizes this provider to the local OCI registry (`wash reg push`)
+
+1. Use `ctl link` inside the `wash up` REPL to establish a link (no values necessary) between the actor and the provider. Note that even if you don't supply configuration values, an actor must be linked to a provider (and have sufficient claims) before it can communicate with it.
```

Comment:
You may have left this out for reasons, but it might be useful to show a few code snippets around the `wash up` repl commands (`ctl link` and `ctl call`). I believe this will help clarify the workflow a bit.

Reply:
Yeah I left them out because I wasn't sure how much that output was going to change over the next couple of days